src/app/page.tsx

```diff
@@ -123,20 +119,17 @@ export default function Home() {
       }
     }
 
-    void fetch("/api/insertQuery", {
+    await fetch("/api/query", {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
       },
       body: JSON.stringify({ query: message }),
-    }).catch((error) => {
```

Comment:
Why did we remove this?

Reply:
Ideally it should have been handled better when this code was originally written. Fetch logs the error even if the catch isn't present. The catch ideally should also update a state to inform the user of an error, since none of it was being done I removed it. Plus API calls should be inside the API folder following Next syntax for APIs